crates/runc-shim/src/console.rs

```diff
@@ -0,0 +1,43 @@
+use std::os::unix::io::RawFd;
```

Comment:
Could you pls add the copyright header to all new files.

Reply:
added